cmd/beacon-chain/slasher-simulator/simulator.go

```diff
@@ -30,6 +32,13 @@ var Commands = &cli.Command{
 	}),
 	Before: func(cliCtx *cli.Context) error {
 		runtime.GOMAXPROCS(runtime.NumCPU())
+
+		logFileName := cliCtx.String(cmd.LogFileName.Name)
```

Comment:
Not a fan of doing stuff in these cli entrypoint files. Can we move this to the actual simulator code?